runtime/src/main/java/mb/pie/runtime/PieChildBuilderImpl.java

```diff
@@ -106,13 +113,21 @@ public PieChildBuilderImpl withExecutorLoggerFactory(Function<Logger, ExecutorLo
         return this;
     }
 
+    @Override public PieChildBuilderImpl withSecondaryParents(Pie... secondaryParents) {
```

Comment:
I'm not really happy with this API. I think it would be better for PieChildBuilderImpl to accept a list of parents. Multiple parents can then be given directly to the constructor, or by calling Pie.createChildBuilder with additional PIE instances.